fdbserver/StorageCache.actor.cpp

```diff
@@ -344,7 +568,7 @@ Key findKey( StorageCacheData* data, KeySelectorRef sel, Version version, KeyRan
 // If the search would depend on any key outside range OR if the key selector offset is too large (range read returns too many bytes), it returns either
 //   a negative offset and a key in [range.begin, sel.getKey()], indicating the key is (the first key <= returned key) + offset, or
 //   a positive offset and a key in (sel.getKey(), range.end], indicating the key is (the first key >= returned key) + offset-1
-// The range passed in to this function should specify a shard.  If range.begin is repeatedly not the beginning of a shard, then it is possible to get stuck looping here
+// The range passed in to this function should specify a cacheRange.  If range.begin is repeatedly not the beginning of a cacheRange, then it is possible to get stuck looping here
```

Comment:
Yes that's correct. I intend to implement the reverse readRange in the next PR. Since we wont ship v1 without it, I just added a TODO in readRange() for now. This is coming shortly basically.